bundles.config.js

```diff
@@ -725,6 +725,13 @@ exports.extensionBundles = [
     latestVersion: '0.2',
     type: TYPES.MISC,
   },
+  {
+    name: 'amp-drilldown',
```

Comment:
Since this is under experiment, and we have an ongoing discussion about naming, let's not block on this as we can find/replace the name once we reach an agreement.